joomla/joomla-cms
# Allow to deprecate WebAssets

### Summary of Changes

Add a posibility to mark asset as Deprecated, with optional description
See https://github.com/joomla/schemas/pull/15


### Testing Instructions

Apply patch.
Edit `media/system/joomla.asset.json`, add to "core" asset:
```
"deprecated": true,
"deprecatedMsg": "The deprecation testing"
```
Enable Debug, and deprecation loging.
Open any page, and check the logs.

### Actual result BEFORE applying this Pull Request
Nothing


### Expected result AFTER applying this Pull Request
There should be an message of deprecated asset "core"


### Link to documentations
Please select:
